### ISSUE TITLE
Crash on Save when no files are opened

### ISSUE THREAD

**Issue by sdurandeu** [DaxStudio/DaxStudio#110]
Repro steps:

1. Open Dax Studio
2. Open File (not a new one, an existing one from disk)
3. Close it (so that there are no open files)
4. Click File -> Save

Maybe it was solved with #94, not sure.

**Comment by dgosbell** [comment]
Yes, this looks like the fix for #94 partially fixes this issue. Although there is a deeper issue here that when you close all the tabs the ribbon was still holding a stale references to the last previously active tab. This has now been fixed and in the next release the Save/Save As options will actually disable when there is no document open.
